addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -1,122 +1,122 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates id="template" xml:space="preserve">
     <t t-name="point_of_sale.OrderReceipt">
+    <!-- ults.is_gcc_country) {
+            results.label_total = _t("TOTAL / اﻹجمالي");
+            results.label_rounding = _t("Rounding / التقريب");
+            results.label_change = _t("CHANGE / الباقي");
+            results.label_discounts = _t("Discounts / الخصومات"); -->
         <div class="pos-receipt p-2">
             <t t-set="showTaxGroupLabels" t-value="doesAnyOrderlineHaveTaxLabel()"/>
-            <ReceiptHeader data="props.data.headerData" />
-            <OrderWidget lines="props.data.orderlines" t-slot-scope="scope"
-                generalCustomerNote="props.data.general_customer_note or ''"
-                internalNote="props.data.internalNote or ''" screenName="props.data.screenName">
+            <ReceiptHeader order="order" />
+            <OrderDisplay order="order" t-slot-scope="scope" mode="'receipt'">
                 <t t-set="line" t-value="scope.line"/>
-                <Orderline basic_receipt="props.basic_receipt" line="omit(scope.line, 'customerNote')" class="{ 'px-0': true }" showTaxGroupLabels="showTaxGroupLabels">
+                <Orderline line="line" showTaxGroup="showTaxGroupLabels" mode="'receipt'">
                     <li t-if="line.customerNote" class="customer-note w-100 p-2 my-1 rounded text-break">
                         <i class="fa fa-sticky-note me-1" role="img" aria-label="Customer Note" title="Customer Note"/>
                         <t t-esc="line.customerNote" />
                     </li>
                 </Orderline>
-            </OrderWidget>
-            <t t-if="!props.basic_receipt">
-                <t t-set="taxTotals" t-value="props.data.taxTotals"/>
```

Comment:
The order_receipt component is now using `taxTotals`, not `getTaxDetails()`. Adapt correctly.